pages/public_cloud/containers_orchestration/managed_kubernetes/forwarding-audit-logs-to-logs-data-platform/guide.en-gb.md

```diff
@@ -0,0 +1,216 @@
+
+Below is an example of an audit log generated by a Kubernetes cluster. *Note the example is not exhaustive.*
+
+```json
+{
+   "kind":"Event",
+   "apiVersion":"audit.k8s.io/v1",
+   "level":"Metadata",
+   "auditID":"XXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX",
+   "stage":"ResponseStarted",
+   "requestURI":"/api/v1/namespaces/searching-worker/configmaps?......",
+   "verb":"watch",
+   "user":{
+      "username":"system:node:search-workers-node-81097c",
+      "groups":[
+         "system:nodes",
+         "system:authenticated"
+      ]
+   },
+   "sourceIPs":[
+      "10.10.0.43"
+   ],
+   "userAgent":"kubelet/v1.26.4 (linux/amd64) kubernetes/f89670c",
+   "objectRef":{
+      "resource":"configmaps",
+      "namespace":"searching-worker",
+      "name":"kube-root-ca.crt",
+      "apiVersion":"v1"
+   },
+   "responseStatus":{
+      "metadata":{
+
+      },
+      "code":200
+   },
+   "requestReceivedTimestamp":"2024-01-01T11:59:06.428015Z",
+   "stageTimestamp":"2024-01-01T11:59:06.428962Z",
+   "annotations":{
+      "authorization.k8s.io/decision":"allow",
+      "authorization.k8s.io/reason":""
+   }
+}
+```
+
+## Instructions
+
+How to enable Audit Log Forwarding
+
+### Enabling Audit Log Forwarding using the OVHcloud Control Panel
+
+#### Create and see existing streams
+
+You can find your streamId, in the Logs Data Platform section of the [OVHcloud control panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB).
+
+- Go to the `Data Stream`{.action} tab of your Logs Data Platform account and `Edit`{.action} the target Data Stream.
+
+https://www.ovh.com/manager/#/dedicated/dbaas/logs/{ldp-account-id}/streams/home
+
+- Copy the `streamId` page of your Logs Data Platform account.
+
+https://www.ovh.com/manager/#/dedicated/dbaas/logs/{ldp-account-id}/streams/{streamid}/edit
+
+### Enabling Audit Log Forwarding using APIs
+
+You will have to define the targeted *Stream* of one of your LDP account on which you want your logs to be forwarded to. The enablement of the forwarding will create a subscription for this stream id.
+Note that the forwarding activation is free of charge, but you will be charged for the usage of the Logs Data Platform service as per standard price plan. For LDP pricing, refer to the [Logs Data Platform product page](https://www.ovhcloud.com/en-gb/logs-data-platform/).
+
+You can retrieve the API specifications in the [OVH API Portal](https://eu.api.ovh.com/console-preview/?section=%2Fdbaas%2Flogs&branch=v1#post-/dbaas/logs/-serviceName-/output/graylog/stream)
+
+#### Step 1 - Retrieve your target Stream (and ID)
+
+List data streams of you Logs Data Platform account:
+
+[API reference](https://eu.api.ovh.com/console-preview/?section=%2Fdbaas%2Flogs&branch=v1#get-/dbaas/logs/-serviceName-/output/graylog/stream)
+
+```shell
+GET /dbaas/logs/{serviceName}/output/graylog/stream
```

Comment:
Why not using our dedicated API calls markdown formats for all the API calls?

Reply:
fixed